core-redirects-studio-plugin/src/main/joo/com/tallence/core/redirects/studio/RedirectManagerStudioPluginBase.as

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Tallence AG
```

Comment:
Hm, die Copyrights müssten nochmal an allen Files gesetzt oder aktualisiert werden, oder? IDEA Copyright Funktion nehmen?
Und vielleicht bei der Gelegenheit ein Link zur Github Seite mit aufnehmen

Reply:
Timo ist alle Java-Klassen noch einmal durchgegangen und ich alle im Studio-Plugin. Wir haben jetzt erstmal den gleichen Copyright Block hinterlegt. Ich denke wir könnten dann in einem separaten PR alle noch einmal aktualisieren und einen Link hinzufügen und / oder die IDEA Funktion nutzen.